app/views/smart_proxies/plugins/_puppet_envs.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if envs.present? %>
+<table class="<%= table_css_classes %> table-fixed">
+  <thead>
+    <th class="col-md-10"><%= _("Environment") %></th>
+    <th class="col-md-2"><%= _("Number of classes") %></th>
+  </thead>
+  <tbody>
+    <% envs.each do |name, count| %>
+      <tr>
+        <td class="ellipsis"><%= name %></td>
```

Comment:
The environment doesn't have a show page, and the form only allows changing the name (which won't affect the proxy anyways) and associating with taxonomies, I don't think this is useful.